hivemind/compression/simple.py

```diff
@@ -0,0 +1,115 @@
+import math
+
+import numpy as np
+import torch
+
+from hivemind.compression.base import Compression, CompressionInfo
+from hivemind.proto import runtime_pb2
+
+
+class NoCompression(Compression):
```

Comment:
I suggest to:

* Move NoCompression to base.py
* Rename this file to float.py

Reply:
[done]